validator/nodejs/postinstall-windows.js

```diff
@@ -20,6 +20,18 @@
 var path = require('path');
 var fs = require('fs');
 
+fs.unlink("postinstall.DELETEME", (err) => {
+	if (err) {
+		console./*OK*/log("Could not delete postinstall.DELETEME. Please delete it.");
+	}
+});
+
+fs.exists('', function(exists) {
```

Comment:
The '' here looks like a leftover / unintentional? Maybe just do the sync version as a best effort here for simplicity. I think it's fine to skip the error logging, in the very rare and unexpected case something goes wrong the function will throw an exception. E.g. this (I haven't tested this yet but we have a Windows box sitting around in another building for gaming ... I promise I will wander over there and see how this works ...):

// On Windows, the script invocation (see package.json's scripts.postinstall line)
// will leave this temporary file behind, so we make an effort to clean it up.
if (fs.existsSync('postinstall.DELETEME')) {
  fs.unlinkSync('postinstall.DELETEME');
}